projects/igniteui-angular/migrations/common/UpdateChanges.ts

```diff
@@ -146,6 +138,32 @@ export class UpdateChanges {
         this.valueTransforms.set(functionName, callback);
     }
 
+    public getQuickInfoAtPosition(entryPath: string, position: number): ts.QuickInfo {
+        const langServ = this.getDefaultLanguageService(entryPath);
+        return langServ.getQuickInfoAtPosition(entryPath, position);
+    }
+
+    public getCompletionsAtPosition(entryPath: string, position: number, options: ts.GetCompletionsAtPositionOptions)
+        : ts.WithMetadata<ts.CompletionInfo> {
+        const langServ = this.getDefaultLanguageService(entryPath);
+        return langServ.getCompletionsAtPosition(entryPath, position, options);
+    }
+
+    public getDefinitionAndBoundSpan(entryPath: string, position: number): ts.DefinitionInfoAndBoundSpan {
+        const langServ = this.getDefaultLanguageService(entryPath);
+        return langServ.getDefinitionAndBoundSpan(entryPath, position);
+    }
+
+    public getSemanticDiagnostics(entryPath: string): ts.Diagnostic[] {
+        const langServ = this.getDefaultLanguageService(entryPath);
+        return langServ.getSemanticDiagnostics(entryPath);
+    }
+
+    public getReferencesAtPosition(entryPath: string, position: number): ts.ReferenceEntry[] {
```

Comment:
Really don't need those exposed as public API for now tbh, the whole point of the update class is to hide the ugly behind a simple `.applyChanges()` call and some jsons :)